test/performance/rrule_test.js

```diff
@@ -43,7 +43,8 @@ perfCompareSuite('rrule', function(perf, ICAL) {
 
     // Apple iCal rules
     "FREQ=MONTHLY;BYDAY=MO,TU,WE,TH,FR;BYSETPOS=3",
-    "FREQ=MONTHLY;BYDAY=SA,SU;BYMONTH=11;BYSETPOS=-1"
+    "FREQ=MONTHLY;BYDAY=SA,SU;BYMONTH=11;BYSETPOS=-1",
+    "FREQ=MONTHLY;BYDAY=3MO,3TU;BYSETPOS=3"
```

Comment:
What is this rule for?

Reply:
`FREQ=MONTHLY;BYDAY=3MO,3TU` returns an array of 2 events [third monday, third tuesday]. `BYSETPOS=3` attempts to grab the 3rd event from the array, which doesn't exist. Without this fix the parser will get stuck in an endless loop looking for the 3rd event. This rule tests that it doesn't get stuck in a loop, without the fix having this rule here makes the tests never return.

Reply:
Ok, it's just testing data. This was the one causing errors?

Reply:
yep